orchestra/todos/import_export.py

```diff
@@ -101,10 +103,11 @@ def import_from_spreadsheet(todo_list_template, spreadsheet_url, request):
             'remove_if': json.loads(row[1] or '[]'),
             'skip_if': json.loads(row[2] or '[]'),
             'slug': row[0] or None,
+            'required': row[3] and row[3].lower() == 'true' or False,
```

Comment:
`or False` it covers a case when there's an empty cell in `row[3]`. That is, if we omit `or False`, it:

- sets True (pythonic true) when it encounters `TRUE` (spreadsheet converts True/true into TRUE) 
- it sets just an empty string when it encounter an empty string

But in case we set `or False`, it sets False to `required`.

Reply:
Please let me know if treating empty strings as False is a bad idea.

Reply:
That was helpful! Thank you! Given it confused me when I read it, I wonder if a more explicit version with two boolean values would be clearer: `isinstance(row[3], str) and row[3].lower() == 'true'` --- what do you think?